devops/ansible/arangodb.yml

```diff
@@ -49,33 +56,7 @@
       shell: ARANGODB_DEFAULT_ROOT_PASSWORD={{ arangodb_root_password }} arango-secure-installation
       become: true
 
-    - name: Copy the SSL certificate files to the server
-      copy:
-        src: '{{ ssl_cert_path }}'
-        dest: /home/ubuntu/
-        local_follow: yes
-        mode: u=rw,g=,o=
-      become: true
-
-    - name: Combine the chain and the private key into one file (required by arango)
-      shell:
-        chdir: /home/ubuntu/
-        cmd: cat fullchain.pem privkey.pem > server.pem
-      become: true
-
-    - name: Set the combined file as 600 permission
-      file:
-        path: /home/ubuntu/server.pem
-        mode: u=rw,g=,o=
-      become: true
-
-    - name: Add an exception to the permissions for the arangodb user
-      shell:
-        chdir: /home/ubuntu/
-        cmd: setfacl -m u:arangodb:r server.pem
-      become: true
-
-    - name: Add the SSL filepath to the config
+    - name: Add the SSL filepath to the config (cert file provided by ssl.yml)
```

Comment:
I'm still slightly confused as to what this block is accomplishing. Where is `/home/ubuntu/server.pem` populated?

Reply:
It happens in the SSL file by manually triggering the shell script (`arango-ssl.sh`).

Reply:
@AlmightyYakob is this ok?